stable/nginx-ingress/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: nginx-ingress
-version: 0.8.13
+version: 0.8.14
```

Comment:
It needs to be higher than the current version, which already is 0.8.14 due to another accepted PR.